crates/sui-indexer-alt-jsonrpc/src/api/checkpoints.rs

```diff
@@ -64,21 +59,12 @@ impl RpcModule for Checkpoints {
 
 /// Load a checkpoint and prepare it for presentation as a JSON-RPC response.
 async fn response(ctx: &Context, seq: u64) -> Result<Checkpoint, RpcError<Error>> {
-    let stored: StoredCheckpoint = ctx
-        .loader()
-        .load_one(CheckpointKey(seq))
+    let (summary, contents, signature) = ctx
```

Comment:
curious what are we gonna do with the duplicate data in PG, have you heard plans?

Reply:
what duplicate data in PG?

Reply:
reading the codes here, we had an impl of data loading from PG, for example here for checkpoint, so I supposed we had the checkpoint data in both PG and KV, which seems duplicate.

Reply:
In production we will only use the KV store (we do not populate the KV tables in the production postgres DB). The PG-based implementations remain to support testing, local network runs and for people who want to run deployments without taking a dependency on Bigtable).